tests/unit/ListNameTranslations.spec.ts

```diff
@@ -12,8 +12,9 @@ import { getVuexStore } from '@/store'
 import { createLocalVue, mount } from '@vue/test-utils'
 
 // Components
-import { ListNameTranslations } from '@/components/YourCompany'
+import { ListNameTranslation } from '@/components/YourCompany'
```

Comment:
Let's all keep in mind that it would be nice to have a "Correction.spec.ts" file where we can test the "resume draft" functionality and verify that all data is read in as expected.

I'm not asking you to do it in this ticket. It should probably be a new (tech debt) ticket.

Reply:
New tech debt ticket for Correction view / save and resume:
https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/5213